llms/mlx_lm/server.py

```diff
@@ -525,9 +522,33 @@ def handle_stream(
             self.wfile.write(f"data: {json.dumps(response)}\n\n".encode())
             self.wfile.flush()
 
+        if self.stream_options["include_usage"]:
```

Comment:
will break if `self.stream_options = None`, right?

Reply:
Thanks for the reminder, I've checked it in new commits.